solidity/test/token_stake/TestTokenStakingEscrow.js

```diff
@@ -16,13 +16,15 @@ const chai = require('chai')
 chai.use(require('bn-chai')(BN))
 const expect = chai.expect
 
-describe('TokenStakingEscrow', () => {
+describe.only('TokenStakingEscrow', () => {
   
   const deployer = accounts[0],
+    grantManager = accounts[0],
```

Comment:
I usually prefer to distinguish accounts for roles tests from the default account. `0` is a default account used for deployment and in most cases we expect the ownership or roles to be transferred to other accounts. Using the default account can give false-positive test results at some point in the future.